src/Maestro/Maestro.Web/Startup.cs

```diff
@@ -549,6 +544,9 @@ public override void Configure(IApplicationBuilder app)
                         ctx.Response.Headers.Add("Referrer-Policy", "no-referrer-when-downgrade");
                     }
 
+                    // Only allow DependencyFlow pages to be rendered on Azure DevOps dashboard
+                    ctx.Response.Headers.Append("Content-Security-Policy", "frame-ancestors https://dev.azure.com");
```

Comment:
Can you add this to `PcsStartup.cs` too? PCS is a new service which Maestro will merge into

Reply:
I ported the changes